ProjectAutomation/ProjectCreationSample/Sdl.ProjectAutomation.ProjectCreationSample.Logic/Sdl.ProjectAutomation.ProjectCreationSample.Logic.csproj

```diff
@@ -16,7 +16,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>$(ProgramFiles)\SDL\SDL Trados Studio\Studio17\</OutputPath>
+    <OutputPath>$(ProgramFiles)\SDL\SDL Trados Studio\Studio18\</OutputPath>

```

Comment:
should this be: $(MSBuildProgramFiles32)?  also in other project files?

Reply:
done